syz-cluster/dashboard/handler.go

```diff
@@ -117,6 +117,10 @@ func (h *dashboardHandler) seriesList(w http.ResponseWriter, r *http.Request) er
 			WithFindings: r.FormValue("with_findings") != "",
 			Limit:        perPage,
 			Offset:       offset,
+			AnyOf: db.SeriesFilterAnyOf{
```

Comment:
It could have been a bit simpler - just a `Name` field, and then `pkg/db` would take care of using it both for patches and series itself.

You anyway fill out both fields with the same value, and I don't think that'll ever change.